hrms/hr/doctype/expense_taxes_and_charges/expense_taxes_and_charges.json

```diff
@@ -106,6 +107,15 @@
   {
    "fieldname": "description_sb",
    "fieldtype": "Section Break"
+  },
+  {
+   "columns": 2,
+   "fieldname": "net_amount",
+   "fieldtype": "Currency",
+   "in_list_view": 1,
+   "label": "Net Amount",
```

Comment:
Where and how does this field get set? Expense taxes and charges are handled by `expense_claim.py`, not erpnext

Reply:
Ah! In this case, this seems a per-analogy errata.

For consistency (and if people ever need to report tax on expense claims), it might make sense to set it in `expense_claim.py`, does it?